chapters/inheritance.tex

```diff
@@ -315,6 +315,7 @@ \section{Modifications}\doublelabel{modifications}
 \end{lstlisting}
 
 {]}
+If a modifier has a description string that overrides the existing description.
```

Comment:
Both description string and documentation string are used for it, but the term "documentation string" is only used in:

> There is also a kind of “documentation comment,” really a documentation string 
> that is part of the Modelica language and therefore not ignored by the Modelica translator.

I would propose to replace that use of "documentation string" as well, since we have the documentation-annotation with a a string as well.

Reply:
Well, I only find one occurrence of _description string_ too.

Reply:
The only variant I really don't like in this context is that use of _documentation comment_, since this might also be interpreted as a comment with some sort of documentation in it.

Reply:
Technically "description string" occurs once, and "description strings" twice (including non-normative ones).
I agree that "documentation comment" is also confusing in that text - but believe we should have a separate ticket for improving that.

Reply:
That separate ticket is #2371